tailwind.config.js

```diff
@@ -3,7 +3,22 @@ module.exports = {
      "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        midnightblue: '#0B1D74',
+        midnightblue2: '#19224D',
```

Comment:
HexCodes from Maria's branding page. I've only added a darker `midnighblue2` for hover effects.